opentelemetry-otlp/tests/integration_test/tests/integration_tests.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(unix)]
```

Comment:
These tests use `use std::os::unix::fs::PermissionsExt` which cannot be used on Windows.